navidrome/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+services:
+  navidrome:
+    image: deluan/navidrome:latest
+    restart: on-failure
+    environment:
+      - PUID=1000
+      - PGID=1000
+    ports:
+      - "4533:4533"
+    volumes:
+      - ${APP_DATA_DIR}/data:/data
+      - ${UMBREL_ROOT}/data/storage/music:/music:rw
+    container_name: navidrome_1
```

Comment:
Is there a specific reason to add the container_name? If not we may remove it :)